example/lib/timestamp.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:floor/floor.dart';
+import 'package:intl/intl.dart';
+
+@Embed()
```

Comment:
To remain constistent with the @entity annotation, can you add a const for this?

```
/// Marks a class as a database entity (table).
const entity = Entity();
```

This way, only one embed object will be created in memory and be reused, which improved memory usage. Of course, it is only minimal,but still a best practice

Reply:
Oh, my bad, I see it already exists even, so please use it